spec/controllers/api/logins_controller_spec.rb

```diff
@@ -32,10 +32,11 @@
       expect(response).to have_http_status(:success)
     end
 
-    it 'returns auth token' do
+    it 'returns auth token and user_id' do
       subject
       json_response = JSON.parse(response.body)
       expect(json_response).to include('auth_token')
+      expect(json_response).to include('user_id')
```

Comment:
Nie da się jednym `include`? Można sprawdzić.

`expect(json_response).to include('auth_token', 'user_id')`